gabbi/httpclient.py

```diff
@@ -55,21 +54,21 @@ def request(self, absolute_uri, method, body, headers, redirect):
 
 
 class VerboseHttp(Http):
-    """A subclass of Http that verbosely reports on activity.
+    """A subclass of ``Http`` which verbosely reports on activity.
```

Comment:
That is correct here as it is required for the meaning of the sentence, see: http://www.writersdigest.com/online-editor/which-vs-that

Reply:
Huh, I'd learned pretty much the opposite - might just be a different school of thought, I don't have a reference handy right now. Reverted back to "that" in f13aa5ee0bc6e5d73f3828126f03cc60a4c5254f.